### DISCUSSION TITLE
Grow Your Local Green Software Community

### DISCUSSION THREAD

**Discussion by NAMRATA-WOKE** [Green-Software-Foundation/discussions#23] [Green Software Meetups]
Meetups have been playing a pivotal role at the base of our movement. These groups provide a platform for like-minded individuals shaping software to come together, share knowledge, exchange ideas, and collaborate on initiatives focused on carbon reduction and clean energy efficiency. Our experts and celebrated advocates will tell you about the inspiration they've been gifted, the meaningful friendships made, and the camaraderie they've felt! 

[Join a local GSF Meetup](https://www.meetup.com/pro/gsf/) or [Grow a local green software community](https://docs.google.com/forms/d/e/1FAIpQLSd8NpRJWexDJt9Qv4P8zK6uKteun6d7Q1w6eUUT_ClQv92RQA/viewform) where you are!

**Comment by AydinMirMohammadi** [comment]
In Germany, our developer community for green software development is growing. We have active groups:
Green Software Development Karlsruhe (monthly meetups)
Green Software Development Manifesto (Quarterly meetups)
Green Coding Berlin 
Green Software Development Stuttgart (starting in September)
hopefully Köln/Bonn starting Q4
maybe Frankfurt (In discussion)

This groups are not part of the GSF Meetup Pro Tennent. An I think, there are more. 

Question/Suggestion: Is it possible to list and visualize the community groups outside of meetup.com? Having our own list/map.

**Reply by adamj89** [reply]
Sorry for the belated answer Aydin. We don't yet have a map but I am adding all our network groups in our wiki: https://grnsft.org/docs/meetup.

Once I have worked out the best way to add a Google map with pins to Confluence, I will give it a go!

**Comment by AydinMirMohammadi** [comment]
Cool. May I ask to add also the URL of the group: https://www.meetup.com/de-DE/green-software-development-karlsruhe/


Von: Adam Jackson ***@***.***>
Gesendet: Dienstag, 25. Juli 2023 09:33
An: Green-Software-Foundation/discussions ***@***.***>
Cc: Aydin Mir Mohammadi ***@***.***>; Comment ***@***.***>
Betreff: [bulk] Re: [Green-Software-Foundation/discussions] Grow Your Local Green Software Community (Discussion #23)


Sorry for the belated answer Aydin. We don't yet have a map but I am adding all our network groups in our wiki: https://grnsft.org/docs/meetup

-
